bundle/config/mutator/process_root_includes.go

```diff
@@ -11,6 +12,17 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+const ExtraIncludePathsKey string = "DATABRICKS_BUNDLE_INCLUDE_PATHS"
```

Comment:
We either should rename it to `DATABRICKS_BUNDLE_INCLUDE_GLOB_PATTERN` or dynamically suffix all paths with `/*.yml`

Reply:
How about `DATABRICKS_BUNDLE_INCLUDES`?